torch/csrc/jit/script/compiler.cpp

```diff
@@ -1446,7 +1508,14 @@ struct to_ir {
     }
     int i = 0;
     for (auto assignee : stmt.lhs()) {
-      if (assignee.kind() == TK_VAR) {
+      if (assignee.kind() == TK_SUBSCRIPT) {
```

Comment:
feels like a copy and paste of the logic above.

Reply:
yeah . . . in general we're not generalizing to the number of assignees well here. I'll try to clean it up